includes/Story_Renderer/Embed.php

```diff
@@ -63,32 +64,53 @@ public function __construct( Story $story ) {
 	 * @return string Rendered block type output.
 	 */
 	public function render( array $args = [] ) {
-		$defaults     = [
+		$defaults = [
 			'align'  => 'none',
 			'height' => 600,
 			'width'  => 360,
 		];
-		$args         = wp_parse_args( $args, $defaults );
-		$align        = sprintf( 'align%s', $args['align'] );
-		$url          = $this->story->get_url();
-		$title        = $this->story->get_title();
-		$poster       = ! empty( $this->story->get_poster_portrait() ) ? esc_url( $this->story->get_poster_portrait() ) : '';
-		$margin       = ( 'center' === $args['align'] ) ? 'auto' : '0';
-		$player_style = sprintf( 'width: %dpx;height: %dpx;margin: %s', absint( $args['width'] ), absint( $args['height'] ), esc_attr( $margin ) );
-		$poster_style = ! empty( $poster ) ? sprintf( '--story-player-poster: url(%s)', $poster ) : '';
 
-		ob_start();
+		$args   = wp_parse_args( $args, $defaults );
+		$align  = sprintf( 'align%s', $args['align'] );
+		$url    = $this->story->get_url();
+		$title  = $this->story->get_title();
+		$poster = ! empty( $this->story->get_poster_portrait() ) ? esc_url( $this->story->get_poster_portrait() ) : '';
+
+		$poster_style  = ! empty( $poster ) ? sprintf( '--story-player-poster: url(%s)', $poster ) : '';
+		$wrapper_style = sprintf(
+			'--aspect-ratio: %F; --width: %dpx; --height: %dpx',
+			$args['height'] / $args['width'],
```

Comment:
Is there an issue with large float numbers. Should be try and limit this to say 3/5 DP?

Reply:
Default here is 6, which I think is fine.

Reply:
I am not sure I understand your answer here. What users can define the height and width. What if I define the height and width as. 609/137 resulting in 4.44525547445

Reply:
Oh, I thought you were talking about the number of decimals for `%F`

This PR doesn't change existing behavior for the embed block's min/max widthheight, users can input whatever they like.

Some sane limits sound reasonable, but that'd be a new ticket.